libpod/define/container_inspect.go

```diff
@@ -506,7 +506,7 @@ type InspectContainerHostConfig struct {
 	// CpuRealtimeRuntime is the length of time (in microseconds) allocated
 	// for realtime tasks within every CpuRealtimePeriod.
 	CpuRealtimeRuntime int64 `json:"CpuRealtimeRuntime"`
-	// CpusetCpus is the is the set of CPUs that the container will execute
+	// CpusetCpus is the set of CPUs that the container will execute
```

Comment:
Might as well move the `on` up a line, that makes the comment line much clearer.